frontend/src/my_team/MyTeamView.jsx

```diff
@@ -99,27 +110,42 @@ function MyTeamView() {
         </div>
       )}
 
-      {!displayScout && recommendations.data && (
-        <button onClick={handleScoutClick}> Scout Opponent</button>
+      {opponents.data && opponents.data.length == 0 && (
+        <Typography>
+          {" "}
+          Add players to opponent team to see scouting view
+        </Typography>
       )}
 
-      {displayScout && recommendations.data && myTeamPlayers.isFetched && (
-        <ScoutOpponent
-          setDisplay={setDisplayScout}
-          myTeamPlayers={myTeamPlayers.data}
-          opponentPlayers={opponents.data}
-          recommendations={recommendations.data.response}
-          refetchPlayers={opponents.refetch}
-          refetchMyTeam={myTeamPlayers.refetch}
-          myTeamStyle={
-            style === ""
-              ? recommendations.data.response.recommendedStyle.style
-              : style
-          }
-          setMyTeamStyle={setStyle}
-        />
+      {!displayScout &&
+        myTeamPlayers.data &&
+        myTeamPlayers.data.length > 0 &&
+        recommendations.data && (
+          <button onClick={handleScoutClick}> Scout Opponent</button>
+        )}
+
+      {displayScout &&
```

Comment:
https://github.com/user-attachments/assets/41c44ebb-261b-4655-989b-599b3c8defbc

If there are no players in MyTeam, there's no point in allowing the user to see any myTeam or scouting, so we make the page empty and prompt the user to add players to their team.